natgeo/mortar
# [AEM-6949] Fixing Broadsheet render issues in IE and some versions of Chrome for Android

https://jira.natgeo.com/browse/AEM-6949

To test:

- Pull AEM-6949
- Run npm test, verify tests/linting all still pass
- Load component in Mortar, and verify layout still intact in Firefox, Chrome, Safari.
- Verify layout now works properly in IE10+
